File: commons/src/main/java/com/ultreon/commons/utilities/python/builtins/Int.java
package com.ultreon.commons.utilities.python.builtins;

import org.jetbrains.annotations.NotNull;

import java.math.BigInteger;

public class Int extends BigInteger {
    public Int(String val, int radix) {
        super(val, radix);
    }

    public Int(@NotNull String val) {
        super(val);
    }
}

File: src/main/kotlin/com/qtech/utilities/python/builtins/Int.kt
package com.qtech.utilities.python.builtins

import java.math.BigInteger
import kotlin.Int

class Int : BigInteger {
    constructor(`val`: String?, radix: Int) : super(`val`, radix) {}
    constructor(`val`: String) : super(`val`) {}

    override fun toByte(): Byte {
        return toInt().toByte()
    }

    override fun toChar(): Char {
        return toInt().toChar()
    }

    override fun toShort(): Short {
        return toInt().toShort()
    }
}